Lib/doctest.py

```diff
@@ -347,6 +361,7 @@ def _comment_line(line):
     else:
         return '#'
 
+
```

Comment:
I think I'd rather we left this file in violation of PEP 8, so that we have a clean diff and it's really clear from the git history exactly which lines we needed to change to get colourised doctests

Reply:
Sure, will revert that commit.